### ISSUE TITLE
[Quest/Loot] Guardian of the Monument (ID: 11059) Loot Table Missing Items

### ISSUE THREAD

**Issue by Vortexsphere** [SunwellTracker/issues#3580]
EDIT:
Killed another guardian today and it dropped the head (so it CAN drop apparently but the drop rate is low, when it should be guaranteed when on the quest). Still no Apexis Crystal, which again, should be guaranteed regardless of being on any quest.

------
------

Decription: 
The quest in Blade's Edge Mountains, "Guardian of the Monument" (for Ogri'la faction), requires you to summon a guardian at the apexis monuments. When you do that (there are 2 methods, either win or lose the minigame, it spawns regardless), the guardian spawns and you kill him. All of that works.

What doesn't work, however, is the correct loot. The guardian is meant to drop 1 x Apexis Crystal (ID: 32572) and, more importantly, the quest item, 1 x Apexis Guardian's Head (ID: 32697). I've killed multiple guardians and not seen one and the drop rate is meant to be 100% (the head only drops when you're on the quest, but the crystal should always drop).

How it works: 
The guardian does not drop the 2 items it should mentioned above. They should be 100% drop rate so don't be mislead by some database entrties. The wowhead comments attest to them dropping when killed and my own memories of retail (I farmed these for completion's sake) attest to them being guaranteed drops. The demons that you also summon for 35 shards seem to work for the crystals at least, so there's that, but it's the quest item I'm more interested in here.

How it should work: 
When killed, the Apexis Guardian should drop 1 x Apexis Crystal and, if you're on the quest, 1 x Apexis Guardian's Head, in addition to other "regular" loot (that part at least seems to work), a chance at a "depleted" item (blue or epic, requires Apexis Shards to make usable), and a small chance at some rare JC recipes (5), apparently only available from this and a select few other mobs that you summon in the zone. I never observed any of this extra loot drop, although I only killed the guardian a few times, but you should check that these are in the loot table as well.

Other items in question:
1) 5 JC recipes: Balanced Shadow Draenite, Great Golden Draenite, Infused Shadow Draenite, Veiled Flame Spessarite, Wicked Flame Spessarite (1% chance drop on each, the loot table for these designs is also part of all the other summoned bosses in the zone, such as the 4 dragons and the demons at the forge camps requiring 35 Apexis Shards to summon)
2) Various "depleted" blues and epics, such as Depleted Cloth Bracers (ID: 32676) and Depleted Dagger (ID: 32673).

Source (you should point out proofs of your report, please give us some source): 
https://www.wowhead.com/quest=11059/guardian-of-the-monument#comments

TL;DR:
The Apexis Guardian should always drop 1 x Apexis Crystal and, more importantly, 1 x Apexis Guardian's Head when someone in the group is on the quest, "Guardian of the Monument".
